CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 Change Log
 ==========
 
-x.x.x
+2.0.0
 -----
 
-* Remove automatic addition of trailing period for LCSH. [#39](https://github.com/unt-libraries/pyuntl/issues/39)
+* Upgraded to python3.
+* Removed automatic addition of trailing period for LCSH. [#39](https://github.com/unt-libraries/pyuntl/issues/39)
+* Removed etd_ms support which is not used by pyuntl.
```

Comment:
Since we are linking to some issues in this Change log, we could add a link on this line to issue #29 and to #18 on the upgrade to python3 line.